addons/binding/org.openhab.binding.nikohomecontrol/META-INF/MANIFEST.MF

```diff
@@ -1,27 +1,29 @@
-Manifest-Version: 1.0
-Bundle-ClassPath: .
-Bundle-ManifestVersion: 2
-Bundle-Name: Niko Home Control Binding
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: org.openhab.binding.nikohomecontrol;singleton:=true
-Bundle-Vendor: openHAB
-Bundle-Version: 2.3.0.qualifier
-Export-Package: 
- org.openhab.binding.nikohomecontrol,
- org.openhab.binding.nikohomecontrol.handler
-Import-Package: 
- com.google.common.collect,
- com.google.gson,
- com.google.gson.stream,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.config.core,
- org.eclipse.smarthome.config.discovery,
- org.eclipse.smarthome.core.library.types,
- org.eclipse.smarthome.core.thing,
- org.eclipse.smarthome.core.thing.binding,
- org.eclipse.smarthome.core.types,
- org.openhab.binding.nikohomecontrol,
- org.openhab.binding.nikohomecontrol.handler,
- org.osgi.framework,
- org.slf4j
-Service-Component: OSGI-INF/*.xml
+Manifest-Version: 1.0
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Niko Home Control Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.nikohomecontrol;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Export-Package: 
+ org.openhab.binding.nikohomecontrol,
+ org.openhab.binding.nikohomecontrol.handler
+Import-Package: 
+ com.google.common.collect,
+ com.google.gson,
+ com.google.gson.stream,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.net,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.openhab.binding.nikohomecontrol,
+ org.openhab.binding.nikohomecontrol.handler,
+ org.osgi.framework,
+ org.osgi.service.component.annotations;version="1.3.0",
```

Comment:
please remove the version constraint and make the import optional

Reply:
Corrected.